mkt/websites/management/commands/import_websites_from_csv.py

```diff
@@ -202,8 +215,65 @@ def parse(self, filename):
         except IOError as err:
             raise CommandError(err)
 
+    def assign_popularity(self):
+        print 'Setting regional popularity values...'
+        for region in self.ranking_per_region.keys():
+            websites_len = len(self.ranking_per_region[region])
+            print u'Setting regional popularity for %d site(s) in %s' % (
+                websites_len, unicode(REGIONS_CHOICES_ID_DICT[region].name))
+            # Sort sites by rank in that region.
+            websites = sorted(self.ranking_per_region[region],
+                              key=itemgetter(1), reverse=True)
+            # Take the same number of popularity values for apps in that
+            # region.
+            apps_popularity = (Installs.objects.filter(region=region)
+                                       .values_list('value', flat=True)
+                                       .order_by('-value')[:websites_len])
```

Comment:
`Installed` is for the users who are logged in, `Installs` is the popularity table for apps. It has this FK:
`addon = models.ForeignKey(Webapp, related_name='popularity')`